java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -2515,10 +2514,8 @@ else if (sa.getStatus().equals(ActionFactory.STATUS_QUEUED)) {
                                 Optional.of(Xor.right(function)), Optional.of(jsonResult))) {
                             LOG.info("Scheduling a package profile update");
 
-                            Action pkgList;
                             try {
-                                pkgList = ActionManager.schedulePackageRefresh(minion.getOrg(), minion);
-                                executeSSHAction(pkgList, minion);
```

Comment:
Out of scope of the PR:

Hi, just to me to understand how things are working, even the SSH Minions, using the scheduleActionExecution will execute the actions on the SSH Minion?
Or this is only for Salt Minion and Traditional clients?

Btw, from Uyuni/Master didn't we removed all about traditional?
I see that here:
https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/manager/action/ActionManager.java#L1171

Reply:
> using the scheduleActionExecution will execute the actions on the SSH Minion?

TL;DR yes. Calling `TaskomatikApi::scheduleActionExecution()` will result in an execution of the `MinionActionExecutor` which in turn (on [line 160](https://github.com/uyuni-project/uyuni/blob/61a6e77c909d84895020a14f0e4d081b9c2d6fe9/java/code/src/com/redhat/rhn/taskomatic/task/MinionActionExecutor.java#L160)) will call `SaltServerActionService::execute()`. This method [splits the minion in regular and ssh](https://github.com/uyuni-project/uyuni/blob/61a6e77c909d84895020a14f0e4d081b9c2d6fe9/java/code/src/com/suse/manager/webui/services/SaltServerActionService.java#L496-L502) and executes the action on both sets. For the latter it will call  `TaskomatikApi::scheduleSSHActionExecution()`

But without knowing the details of all of these, I'm wondering if you have a point and all this back and forth is actually needed. Probably @lucidd knows better.

> Uyuni/Master didn't we removed all about traditional?

Not sure what's the state of this but it seems to me we have multiple places that still refers to traditional stack on master. @mcalmer is this expected to be already removed?

Reply:
I mostly dropped the client side. Removing all the code from the server side needs much more time and is potential dangreous. There is still a lot of time we can spend in removing traditional stack from Uyuni.

Reply:
> But without knowing the details of all of these, I'm wondering if you have a point and all this back and forth is actually needed. Probably @lucidd knows better.

There is a lot of moving parts and i guess technically not all of that is needed if you already know you are working with salt ssh. But i think for now its better if everything goes though the same codepath for executing actions otherwise we risk the chance of having code introduced in on path but not the other.

Reply:
Thanks for all the info!